packages/core/src/components/advanced-select/advanced-select.tsx

```diff
@@ -0,0 +1,160 @@
+import { Component, ComponentInterface, Prop, h, FunctionalComponent, Event, EventEmitter } from "@stencil/core";
+
+import {
+  AdvancedSelectGroupRedirect,
+  AdvancedSelectOption,
+  AdvancedSelectOptionsOrGroup,
+} from "./advanced-select.models";
+import {
+  AdvancedSelectClickEvent,
+  AdvancedSelectOptionClickEvent,
+  AdvancedSelectRedirectClickEvent,
+} from "./advanced-select.interfaces";
+
+@Component({
+  tag: "dso-advanced-select",
+  styleUrl: "advanced-select.scss",
+  shadow: true,
+})
+export class AdvancedSelect implements ComponentInterface {
+  /**
+   * The options to display in the select.
+   */
+  @Prop()
+  options: AdvancedSelectOptionsOrGroup<any>[] = [];
+
+  /**
+   * The active option. By object reference.
+   */
+  @Prop()
+  active?: AdvancedSelectOption<any>;
+
+  /**
+   * An extra text for the active option. Only visible in the list of options.
+   */
+  @Prop()
+  activeHint?: string;
+
+  /**
+   * The open state of the options list.
+   */
+  @Prop()
```

Comment:
Deze graag reflecteren naar het DOM met `@Prop({ reflect: true })`.

In principe doen we dit voor alle simpele/korte types. `activeHint` zou ook kunnen, maar daar mag jij over nadenken ;-)

Reply:
Persoonlijk zie ik de meerwaarde van `reflect` niet, maar ik kan mij nog wel (ergens in de verte) voorstellen dat het voor `open` handig is om te doen. Voor `activeHint` zie ik dat dan weer totaal niet omdat het een `string` is wat vanalles kan bevatten.

Heb het voor `open` toegevoegd.